packages/truffle/test/scenarios/solidity_testing/solidityTests.js

```diff
@@ -47,6 +47,7 @@ describe("Solidity Tests", function () {
           assert(output.includes("1 passing"));
         })
         .catch(error => {
+          console.log("the output is -- %o", logger.contents());
```

Comment:
This should be removed or put behind a `debug` log.

Reply:
I've started putting these things in my tests. Like if the test fails (as is implied since we are in the `catch`), we will want to see what the output is to debug it. Is it better practice to use `debug`?

Reply:
Oh wait, it's a test. My bad! this is fine to leave like this as it makes debugging easier!